libraries/AP_HAL_ChibiOS/hwdef/AnyleafH7/defaults.parm

```diff
@@ -0,0 +1,23 @@
+# Onboard ELRS
+SERIAL5_PROTOCOL 23 # RCIN
+RC_OPTIONS 8192  # Set 420k baud for ELRS
+RC5_OPTION 154 # Set up arm switch to `ArmDisarm with AirMode`
+RC2_REVERSED 1 # Prevent inverse pitch controls
+RSSI_TYPE 3 # RC provided RSSI
+ARMING_RUDDER 0  # Disable rudder arming
+
+
+# Onboard OSD
+OSD_TYPE 5 # MSP
+SERIAL2_PROTOCOL 42 # DisplayPort
+OSD_CELL_COUNT 0 # auto based on voltage at start
+
+
+# ESC telemetry
+SERIAL3_PROTOCOL 16 # ESC telemetry
+
+
+# Remove default serial protocols; set to None.
+SERIAL1_PROTOCOL -1
```

Comment:
remove this...it should be telem if not used for DJI SBUS

Reply:
Interesting re defaults. Of note, I think CAN is probably a better bet for both GPS and telem. (And you can do telem with the ELRS  onboard as well), but this should be harmless for UART devices.